API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
OK, I've worked it out with @Marsup  on gitter.

I'm closing this for now until I figure out how to better word it. One problem for me was that "`[]` notation" isn't defined anywhere, so I thought it meant something different.